packages/ember-metal/lib/computed_macros.js

```diff
@@ -648,11 +648,11 @@ export function readOnly(dependentKey) {
 export function deprecatingAlias(dependentKey) {
   return computed(dependentKey, {
     get(key) {
-      Ember.deprecate(`Usage of \`${key}\` is deprecated, use \`${dependentKey}\` instead.`);
+      Ember.deprecate(`Usage of \`${key}\` is deprecated, use \`${dependentKey}\` instead.`, false, { id: 'ember-metal.deprecating-alias', until: '3.0.0' });
```

Comment:
This one needs to be tweaked a bit. We need to provide options at the calling site. 

Basically, all usage of deprecatingAlias need to be updated, so that the 'id' and 'until' is unique to each deprecated property.